src/style.css

```
@@ -0,0 +1,133 @@
+* {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+/* #region .card */
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-bottom: 17px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px;
+}
+
+.card__title-text {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  list-style-type: none;
+  padding: 8px 16px;
+}
+
+.card__title-text_small {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding-top: 4px;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 16px;
+  padding-bottom: 0;
```

but you can reset all paddings on reset file (like margins)

```suggestion
  padding: 16px 0 0;
```